docs/FilteringTutorial.md

```diff
@@ -88,24 +88,7 @@ Elements passed as `filters` are regular inputs. That means you can build sophis
 
 ### `<SearchInput>`
 
-<video controls autoplay playsinline muted loop>
-  <source src="./img/search_input.webm" type="video/webm"/>
-  <source src="./img/search_input.mp4" type="video/mp4"/>
-  Your browser does not support the video tag.
-</video>
-
-
-In addition to [the usual input types](./Inputs.md) (`<TextInput>`, `<SelectInput>`, `<ReferenceInput>`, etc.), you can use the `<SearchInput>` in the `filters` array. This input is designed especially for the [`Filter Form`](./FilterForm.md). It's like a `<TextInput resettable>` with a magnifier glass icon - exactly the type of input users look for when they want to do a full-text search.
-
-```jsx
-import { SearchInput, TextInput } from 'react-admin';
-
-const postFilters = [
-    <SearchInput source="q" alwaysOn />
-];
-```
-
-In the example given above, the `q` filter triggers a full-text search on all fields. It's your responsibility to implement the full-text filtering capabilities in your `dataProvider`, or in your API.
+To enable users to search for data in the table, you can add the `<LiveSearchInput>` component to the `filters` array of your `<List>` as explained in the component chapter.
```

Comment:
Why did you remove the previous content? I think we should keep it.

Did you mean `<SearchInput>` instead of `<LiveSearchInput>`?

Lastly, instead of asking to go see the 'component chapter', why not provide a link to it?

Reply:
Contrary to the code, for documentation, it's not necessarily a bad thing to repeat yourself. It improves the discoverability of the doc (the same piece of information can be found at multiple places) and also minimizes the number of clicks necessary to access the information.

In this particular case I'd meet halfway: keep the previous paragraph, which will serve as an introduction to the component, and also include a link to the component's dedicated page, for those who want to go further in the explanations.